components/Profile/SupProfileHeader.js

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import {
+  View,
+  Text,
+  TouchableOpacity,
+  Linking,
+  ImageBackground,
+  Button
+} from "react-native";
+import { ScrollView } from "react-navigation";
+import * as WebBrowser from "expo-web-browser";
+import { Avatar } from "react-native-elements";
+import SvgUri from "react-native-svg-uri";
+
+import styles from "../../constants/Profile/SupProfileHeader";
+import traschCan from "../../constants/DetailScreen/DetailHeader";
+//icons
+import Envelope from "../../assets/jsicons/socialmedia/Envelope";
+import Instagram from "../../assets/jsicons/socialmedia/Instagram";
+import Twitter from "../../assets/jsicons/socialmedia/Twitter";
+import Facebook from "../../assets/jsicons/socialmedia/Facebook";
+import { randomImage } from "../Animals/RandomImage";
+import MapMarker from "../../assets/jsicons/headerIcons/map-marker";
+import Connect from "../../components/Connect";
+
+const SupProfileHeader = props => {
+  let profile = props.profile;
+  const randomHeaderImage = randomImage();
+  return (
+    <ScrollView style={styles.pic}>
+      <ImageBackground
+        source={randomHeaderImage}
+        style={{
+          backgroundColor: "#000",
+          paddingTop: 100
+        }}
+        imageStyle={{ opacity: 0.7 }}
+      >
+        <View style={styles.container}>
+          <View style={styles.allContainer}>
+            <View style={styles.leftContainer}>
+              <Avatar
+                size={100}
+                rounded
+                source={{
+                  uri: profile.profile_image
+                }}
+              />
+            </View>
+
+            <View style={styles.rightContainer}>
+              <View style={styles.textContainer}>
+                <View style={traschCan.nameContainer}>
+                  <Text style={styles.titleText}>{profile.sup_name}</Text>
+                </View>
+                <Text style={styles.userText}>
+                  <MapMarker />
+                  {profile.location}
+                </Text>
+                {/* <Text style={styles.userText}>@{profile.username}</Text> */}
+              </View>
+              {/* <View style={styles.socialContainer}>
+							<TouchableOpacity
```

Comment:
Can we remove all this commented out code? No reason to keep it

Reply:
This file should no longer exist, please confirm no work has been done here that needs to be considered, otherwise this file has been deleted and should be removed from this branch before merging as well